src/discord-update-report.js

```diff
@@ -6,7 +6,7 @@ function formatReport(report) {
   out.push('');
   out.push('**CATALOG UPDATE**');
   for (const r of report) {
-    out.push(`- ${r.catalog} :`);
+    out.push(`- [${r.catalog}](https://keycap-archivist.com/maker/${r.catalog.toLowerCase().replace(/\s+/g, "-")}) :`);
```

Comment:
Not sure if we'll have difference with the makers name but if you want to be tight to the generated slugs in website please refer to: https://github.com/keycap-archivist/website/blob/91a3565a29ea034de227dd50dc74b950e4ec9825/gatsby-node.js#L16

Reply:
If you mean embedded previews as in when you link a website and discord shows the embed meta from the site... then no, by default.. Discord hyperlinks do not display embeds beneath them.. As if they did, this pull request ( if approved ) would make catalog updates look horrendous.. and you'd have to scroll up to see the majority of the catalog changes..

Reply:
I'll see about the slugify module.. it seems that'll be helpful..